docs-src/descriptions/one-source.pug

```diff
@@ -829,8 +829,25 @@ pre(title='events in time').
                  12      14
 div
 
++descr-method('thru', 'thru', 'obs.thru(callback)').
+  This transformation method takes a single callback and calls the callback with
+  #[b obs] observable. Based on the most.js method of the same name, this provides
```

Comment:
Maybe describe it like "... this allows to integrate your helper functions to chains of Kefir method calls without adding them to Observable prototype"? And then in the example show something like this `source.map(...).thru(callback).filter(...)`? I just think we should mention chaining use case explicitly. 

Also I'd call the argument `transformer` instead of `callback`. And maybe mention in the description that `o.thru(fn)` is equivalent to `fn(o)`, I think this should immediately click in reader's mind what that method does.